chapter_07_aggregate.asciidoc

```diff
@@ -529,6 +529,11 @@ the `allocations` tables, we force both of them to also try and update the
 `version_number` in the `products` table, in such a way that only one of them
 can win and the world stays consistent.
 
+TIP: This is optimistic concurrency control. It's optimistic because we assume
+everything will be _fine_ when two users want to make changes to the database.
+Pessimistic concurrency control assumes that two users will cause conflicts, so
+we lock everything just to be safe.
```

Comment:
we're stacking up a lot of tips and notes in this section of the chapter.  imma have a go at putting all these into a sidebar...